hack/make/dependencies.make

```diff
@@ -92,3 +94,7 @@ clusteradm: $(CLUSTERADM)
 $(CLUSTERADM):
 	test -s $(LOCALBIN)/clusteradm || curl -sL https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | INSTALL_DIR=bin bash -s -- $(CLUSTERADM_VERSION)
 
+.PHONY: subctl
```

Comment:
im getting a 404 error for subctl when its trying to download it

Reply:
compared the url against if you were to download manually and it looks the same i wonder is there a flag with curl that you need

Reply:
I see this issue too now but only in iterm.
I didn't see it in vscode terminal.
The value for uname -m is coming back as x86 vs arm
So it's a Mac issue with that installer script as the arm version of the cli doesn't exist.
We should create an issue upstream for it.

If you want a workaround on Mac, you can download the amd mac binary and extract it to the dir. The make target shouldn't try to redownload if it's already there